tests/templates/test_subroutines/test_interferometer.py

```diff
@@ -20,6 +20,13 @@
 from pennylane.wires import Wires
 
 
+def expected_shapes(n_wires):
```

Comment:
I agree. Again, I used this method because it was used elsewhere. So, I wanted to keep continuation.
I will change this as your method makes more sense.